warehouse/models/staging/gtfs/stg_gtfs_schedule__pathways.sql

```diff
@@ -0,0 +1,29 @@
+WITH external_pathways AS (
+    SELECT *
+    FROM {{ source('external_gtfs_schedule', 'pathways') }}
+),
+
+stg_gtfs_schedule__pathways AS (
+
+    -- Trim all string fields
+    -- Incoming schema explicitly defined in gtfs_schedule_history external table definition
+
+    SELECT
+        base64_url,
+        ts,
+        {{ trim_make_empty_string_null('pathway_id') }} AS pathway_id,
+        {{ trim_make_empty_string_null('from_stop_id') }} AS from_stop_id,
+        {{ trim_make_empty_string_null('to_stop_id') }} AS to_stop_id,
+        pathway_mode,
```

Comment:
Let's maybe add a comment here noting that these have numeric types set in the external table, just to explain why it's inconsistent with the other tables (not being trimmed / cast)?